packages/database/src/migrations/20210604004937-AddNewNationalLevelCovidTrackingByGenderDashboard-modifies-data.js

```diff
@@ -0,0 +1,141 @@
+};
+
+const OLD_DASHBOARD_ID = 'COVID_TRACKING_By_Gender_by_district';
+
+const FJ_DASHBOARD_ID = 'FJ_COVID_TRACKING_By_Gender_Country';
+const NR_DASHBOARD_ID = 'NR_COVID_TRACKING_By_Gender_Country';
+
+const FJ_DASHBOARD_GROUP_CODE = 'FJ_Covid_Fiji_Country_COVID-19';
+const NR_DASHBOARD_GROUP_CODE = 'NR_Covid_Nauru_Country_COVID-19';
+
+const DASHBOARD_OBJECT = {
+  dataBuilder: 'composeDataPerOrgUnit',
+  dataBuilderConfig: {
+    dataBuilders: {
+      Males: {
+        dataBuilder: 'sumByOrgUnit',
+        dataBuilderConfig: {
+          aggregationType: 'FINAL_EACH_DAY',
+          dataElementCodes: ['COVIDVac5'],
+          entityAggregation: {
+            dataSourceEntityType: 'facility',
+            aggregationEntityType: 'country',
+          },
+        },
+      },
+      Total: {
+        dataBuilder: 'composeDataPerOrgUnit',
+        dataBuilderConfig: {
+          dataBuilders: {
+            Males: {
+              dataBuilder: 'sumByOrgUnit',
+              dataBuilderConfig: {
+                aggregationType: 'FINAL_EACH_DAY',
+                dataElementCodes: ['COVIDVac5'],
+                entityAggregation: {
+                  dataSourceEntityType: 'facility',
+                  aggregationEntityType: 'country',
+                },
+              },
+            },
+            Females: {
+              dataBuilder: 'sumByOrgUnit',
+              dataBuilderConfig: {
+                aggregationType: 'FINAL_EACH_DAY',
+                dataElementCodes: ['COVIDVac6'],
+                entityAggregation: {
+                  dataSourceEntityType: 'facility',
+                  aggregationEntityType: 'country',
+                },
+              },
+            },
+          },
+        },
+      },
+      Females: {
+        dataBuilder: 'sumByOrgUnit',
+        dataBuilderConfig: {
+          aggregationType: 'FINAL_EACH_DAY',
+          dataElementCodes: ['COVIDVac6'],
+          entityAggregation: {
+            dataSourceEntityType: 'facility',
+            aggregationEntityType: 'country',
+          },
+        },
+      },
+    },
+  },
+  viewJson: {
+    name: 'Number of people fully vaccinated for COVID-19 (by sex)',
+    type: 'chart',
+    chartType: 'bar',
+    chartConfig: {
+      Males: {
+        stackId: 1,
+      },
+      Females: {
+        stackId: 2,
+      },
+    },
+    periodGranularity: 'day',
+  },
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+    UPDATE "dashboardGroup"
+    SET "dashboardReports" = array_remove("dashboardReports", '${OLD_DASHBOARD_ID}')
+    WHERE code IN (${arrayToDbString([FJ_DASHBOARD_GROUP_CODE, NR_DASHBOARD_GROUP_CODE])});
+  `);
+
+  await db.runSql(`
+    DELETE FROM "dashboardReport" 
+    WHERE id =  '${OLD_DASHBOARD_ID}';
+  `);
+
+  await insertObject(db, 'dashboardReport', {
+    id: FJ_DASHBOARD_ID,
+    ...DASHBOARD_OBJECT,
+  });
+
+  await insertObject(db, 'dashboardReport', {
+    id: NR_DASHBOARD_ID,
+    ...DASHBOARD_OBJECT,
+  });
```

Comment:
A bit of context: this dashboard was used in both FJ and Nauru before. But we've run into situation where the data level is changed in this country, but for the other country it should stay the same. So even though the data level are the same at the moment between Nauru and Fiji, I've splitted them into 2 separate dashboards for each country. So that when there's data for Nauru, if we decide it's not going to have the same data level as Fiji, we can change it easily and won't affect the existing dashboard in Fiji.

Also the same like above, changing the dataSourceEntityType and aggregationEntityType here